recipes/astra-toolbox/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set version = "2.0.0" %}
+
+package:
+  name: astra-toolbox
+  version: {{ version }}
+
+source:
+  url: https://github.com/astra-toolbox/astra-toolbox/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 214361cb02b61194a443c7816b2d37a10e134c68789e7eacc931354a6e8ecc97
+
+build:
+  number: 0
+  # libastra is only available for these platforms presently:
+  skip: true  # [not (linux64 or win64)]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+  host:
+    - cython >=0.13
+    # libastra includes boost in its public headers
+    - boost-cpp
+    - libastra =={{ version.split('.')[:2]|join('.') }}
```

Comment:
Is there a reason libastra doesn't have a run export?

Reply:
The libastra recipe does have a run_export, but run_exports don't pin in the host section, they pin in the run section.